libs/dh/market-participant/actors/feature-delegation/src/lib/create/dh-delegation-create-modal.component.html

```diff
@@ -43,7 +43,16 @@
       <watt-datepicker-v2
         [label]="t('start')"
         [formControl]="createDelegationForm.controls.startDate"
-      />
+        [min]="date"
+      >
+        <watt-field-error
+          *ngIf="
```

Comment:
suggestion: let's use the new `@if` syntax

Reply:
fixed